pkg/controllers/restore/controller.go

```diff
@@ -360,6 +360,12 @@ func (h *handler) generateDependencyGraph(ownerToDependentsList map[string][]res
 		name := resourceInfo.Name
```

Comment:
`// add to adjacency list` comment was already somewhat out of place and it's even more so now after we added skip logic so close to it. Suggesting to remove this comment as it's largely redundant.

Reply:
removed.